[Device-capability] [Create and update document][1452622] (#2656)
MicrosoftDocs/msteams-docs: msteams-platform/concepts/device-capabilities/device-capabilities-overview.md

## Error handling

The following table lists the error codes and under what conditions they are generated:

|Error code |  Error Name     | Condition|
| --------- | --------------- | -------- |
| **100** | NOT_SUPPORTED_ON_PLATFORM | API is not supported on the current platform.|
| **404** | FILE_NOT_FOUND | File specified is not found in the given location.|
| **500** | INTERNAL_ERROR | Internal error is encountered while performing the required operation.|
| **1000** | PERMISSION_DENIED |Permission is denied by the user.|
| **2000** |NETWORK_ERROR | Network issue.|
| **3000** | NO_HW_SUPPORT | Underlying hardware doesn't support the capability.|
| **4000**| INVALID_ARGUMENTS | One or more arguments are invalid.|
| **5000** | UNAUTHORIZED_USER_OPERATION | User is not authorized to complete this operation.|
| **6000** |INSUFFICIENT_RESOURCES | Operation couldn't be completed due to insufficient resources.|

> |**7000** | THROTTLE | Platform throttles the request because the API is invoked too frequently.|

Reviewer: Suggestion

```suggestion
|**7000** | THROTTLE | Platform throttled the request as the API was invoked frequently.|
```

|  **8000** | USER_ABORT |User aborts the operation.|

> | **9000**| OLD_PLATFORM | Platform code is outdated and doesn't implement this API.|

Reviewer: For consistency, please change to:
```suggestion
| **9000**| OLD_PLATFORM | Platform code is outdated and does not implement this API.|
```
